### ISSUE TITLE
Computation of climatologies via functions

### ISSUE THREAD

**Issue by pwolfram** [MPAS-Dev/MPAS-Analysis#114]
Climatology calculations, e.g.,  

```python

    # make a dictionary of the months in each climotology
    monthsInClim = {}
    monthsInClim['winNH'] = [1, 2, 3]
    monthsInClim['sumNH'] = [7, 8, 9]
    monthsInClim['winSH'] = [12, 1, 2]
    monthsInClim['sumSH'] = [6, 7, 8]
    monthsInClim['on'] = [10, 11]
    monthsInClim['fm'] = [2, 3]

... 
    # check that each year has 24 months (?)
    monthlyClimotology = dsTimeSlice.groupby('Time.month').mean('Time')
    daysInMonth = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    monthLetters = ['J', 'F', 'M', 'A', 'M', 'J', 'J', 'A', 'S', 'O', 'N', 'D']

    climatologies = {}
    for climName in monthsInClim:
        months = monthsInClim[climName]
        month = months[0]
        days = daysInMonth[month-1]
        climatology = days*monthlyClimotology.sel(month=month)
        totalDays = days
        for month in months[1:]:
            days = daysInMonth[month-1]
            climatology += days*monthlyClimotology.sel(month=month)
            totalDays += days
        climatology /= totalDays

        climatologies[climName] = climatology
```

should be converted into functions for production of monthly climatologies, e.g., at https://github.com/MPAS-Dev/MPAS-Analysis/blob/develop/mpas_analysis/sea_ice/modelvsobs.py#L192

**Comment by xylar** [comment]
Should be addressed by #117

**Comment by xylar** [comment]
Closed via #117
